### ISSUE TITLE
Validation schemas don’t work (when offline?)

### ISSUE THREAD

**Issue by luis100** [keeps/roda#240]
```
2016-03-08 14:19:29,869 [http-nio-8080-exec-4] ERROR o.r.c.c.validation.ValidationUtils - Error validating descriptive metadata
org.xml.sax.SAXParseException: src-resolve: Cannot resolve the name 'xml:lang' to a(n) 'attribute declaration' component.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.util.ErrorHandlerWrapper.error(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.reportSchemaError(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.reportSchemaError(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.getGlobalDecl(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDAttributeTraverser.traverseLocal(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDAbstractTraverser.traverseAttrsAndAttrGrps(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.processComplexContent(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.traverseComplexContent(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.traverseComplexTypeDecl(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.traverseGlobal(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.traverseSchemas(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.parseSchema(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.XMLSchemaLoader.loadSchema(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.XMLSchemaLoader.loadGrammar(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.impl.xs.XMLSchemaLoader.loadGrammar(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at org.apache.xerces.jaxp.validation.XMLSchemaFactory.newSchema(Unknown Source) ~[xercesImpl-2.11.0.jar:na]
	at javax.xml.validation.SchemaFactory.newSchema(SchemaFactory.java:638) ~[na:1.8.0_51]
	at org.roda.core.common.validation.ValidationUtils.validateDescriptiveBinary(ValidationUtils.java:225) ~[roda-core-2.0.0-SNAPSHOT.jar:na]
	at org.roda.wui.api.controllers.BrowserHelper.updateDescriptiveMetadataFile(BrowserHelper.java:743) [classes/:na]
	at org.roda.wui.api.controllers.Browser.updateDescriptiveMetadataFile(Browser.java:595) [classes/:na]
	at org.roda.wui.server.browse.BrowserServiceImpl.updateDescriptiveMetadataFile(BrowserServiceImpl.java:251) [classes/:na]
```

**Comment by jmaferreira** [comment]
I believe this happened with RODA-in as well. Use the same XSD of that project.

**Comment by hsilva-keep** [comment]
@luis100 Please provide example data where this problem occurs.

**Comment by hsilva-keep** [comment]
First thing first: the problem indeed is related to XML validation when no network connection is available. 
Secondly, it happens when validating Dublin Core. And it happens because there is an import for the XML namespace as follows (to be able to reference xml:lang element):
```xml
<xs:import namespace="http://www.w3.org/XML/1998/namespace"
    schemaLocation="file://www.w3.org/2001/03/xml.xsd">
</xs:import>
```
Downloading the XML schema e changing the above declaration for a local URI does nothing as XERCES still complain.

@jmaferreira actually RODA-in also faces the same problem. Just tested it and here's the output:
```
2016-03-16 14:59:07,377 [Thread-11] INFO  org.roda.rodain.utils.Utils - Error validating the XML with the EAD schema
org.xml.sax.SAXParseException: src-resolve: Cannot resolve the name 'xml:lang' to a(n) 'attribute declaration' component.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.util.ErrorHandlerWrapper.error(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.reportSchemaError(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.getGlobalDecl(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDAttributeTraverser.traverseLocal(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDAbstractTraverser.traverseAttrsAndAttrGrps(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.processComplexContent(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.traverseComplexContent(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.traverseComplexTypeDecl(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDComplexTypeTraverser.traverseGlobal(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.traverseSchemas(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.traversers.XSDHandler.parseSchema(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.XMLSchemaLoader.loadSchema(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.XMLSchemaLoader.loadGrammar(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.impl.xs.XMLSchemaLoader.loadGrammar(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.apache.xerces.jaxp.validation.XMLSchemaFactory.newSchema(Unknown Source) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at javax.xml.validation.SchemaFactory.newSchema(SchemaFactory.java:638) ~[na:1.8.0_74]
	at org.roda.rodain.utils.Utils.validateSchemaWithoutCatch(Utils.java:100) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.roda.rodain.utils.Utils.validateSchema(Utils.java:83) ~[rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.roda.rodain.inspection.InspectionPane$1.call(InspectionPane.java:179) [rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at org.roda.rodain.inspection.InspectionPane$1.call(InspectionPane.java:170) [rodain-app-2.0.0-alpha.3.1(2).jar:na]
	at javafx.concurrent.Task$TaskCallable.call(Task.java:1423) [jfxrt.jar:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_74]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_74]
```

**Comment by hsilva-keep** [comment]
Fixed in cc645f68521aa004dfaecfd96d63d4e59279fc1e.
